app/Models/Course.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Course extends Model
+{
+    use HasFactory, SoftDeletes;
+
+    protected $table = 'courses';
+
+    protected $fillable = [
+        'title',
+        'logo_path',
+        'introduction',
+        'description',
+        'learn_time',
+        'quizzes',
+        'price',
+    ];
+
+    public function users()
+    {
+        return $this->belongsToMany(Users::class, 'course_users');
+    }
+
+    public function lesson()
```

Comment:
có câu hỏi là `$this->hasMany(Lesson::class, 'course_id');` sẽ trả về cái gì em nhỉ?

Reply:
trả về trả về mối quan hệ 1-n của lesson với course_id với course_id là khóa ngoại ạ

Reply:
vậy nó sẽ trả về 1 lesson hay nhiều lesson?

Reply:
nhiều lesson ạ

Reply:
@hapo-binhpt nếu là nhiều lesson thì tên function đang sai nhé, phải là `lessons` có `s` nhé
em sửa các phần tương tự